src/data/services/EnterpriseAccessApiService.js

```diff
@@ -9,9 +9,7 @@ class EnterpriseAccessApiService {
 
   static getSubsidyRequestConfiguration({ enterpriseId, clearCacheEntry = false }) {
     const url = `${EnterpriseAccessApiService.baseUrl}/customer-configurations/${enterpriseId}/`;
-    return EnterpriseAccessApiService.apiClient({
-      useCache: configuration.USE_API_CACHE,
-    }).get(url, { clearCacheEntry });
+    return EnterpriseAccessApiService.apiClient().get(url, { clearCacheEntry });
```

Comment:
[curious] If this API call never uses `useCache` anymore, is the `clearCacheEntry` logic necessary anymore in this function's signature, passed to this `get` function, or in the caller(s) of `getSubsidyRequestConfiguration`?